janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -317,13 +317,14 @@ private static CreateTableWithOptions compactionOptions(final CreateTableWithOpt
                 Case($("SizeTieredCompactionStrategy"), sizeTieredCompactionStrategy()),
                 Case($("TimeWindowCompactionStrategy"), timeWindowCompactionStrategy()),
                 Case($("LeveledCompactionStrategy"), leveledCompactionStrategy()));
-        Iterator<Array<String>> groupedOptions = Array.of(configuration.get(COMPACTION_OPTIONS))
-            .grouped(2);
 
-        for(Array<String> keyValue: groupedOptions){
-            compactionStrategy = compactionStrategy.withOption(keyValue.get(0), keyValue.get(1));
+        if (configuration.has(COMPACTION_OPTIONS)) {
```

Comment:
I just ran the test locally. Seems that this expression evaluates to false in the test, causing the test coverage to be 0.

Reply:
I'll add a test with COMPACTION_OPTIONS set then 👍